x-pack/plugins/reporting/server/lib/content_stream.ts

```diff
@@ -78,60 +85,20 @@ export class ContentStream extends Duplex {
 
   constructor(
     private client: ElasticsearchClient,
-    private exportTypesRegistry: ExportTypesRegistry,
     private logger: LevelLogger,
-    private document: ContentStreamDocument
+    private document: ContentStreamDocument,
+    { encoding = 'base64' }: ContentStreamParameters = {}
```

Comment:
Nitpick: can we make `logger` the final function argument? That would make it consistent with the rest of the Reporting code. I think it helps order the parameters from most critical to the least critical.

Reply:
This is a good point, and since my point was a nitpick, let's move forward with how you have it.